meteor_packages/mats-common/package.js

```diff
@@ -104,9 +104,8 @@ Package.onUse(function (api) {
     api.addFiles('imports/startup/client/index.js');
     api.addFiles('imports/startup/client/init.js');
     api.addFiles('imports/startup/client/routes.js');
-    //api.addFiles('imports/startup/client/jspdf.js');
```

Comment:
Is this the only change we wanted to make? I'm guessing we want to do this because `imports/startup/client/jspdf.js` doesn't exist?